basic.css

```diff
@@ -0,0 +1,86 @@
+html {
+  height: 100%;
+}
+
+body {
+  min-height: 100%;
+}
+
+.toolbar {
+  background-color: #3c3a39;
+  margin: 0%;
+  padding: 12px 24px;
+  height: fit-content;
+}
+
+.dropdown {
+  margin-left: auto;
+}
```

Comment:
- [ ] Kindly make the hamburger menu icon be visible just as the one in the Figma design.
- **Your design**
![image](https://user-images.githubusercontent.com/50941074/164762421-e6cb558f-de2e-4870-bd29-6816d4cf617a.png)

- **The Figma Design**
![image](https://user-images.githubusercontent.com/50941074/164762511-c692a206-a5c9-40c3-925c-212c01abff94.png)